packaging/kubernetes/openshift-workload-pinning

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
How is it configured in openshift?

Reply:
On versions prior to 4.12 and earlier, this was configured by MCO with the customer defining these configurations or we would via things like ZTP. Going forward from 4.13, this is now controlled by NTO.